packages/sku/lib/suggestScript.js

```diff
@@ -42,28 +32,24 @@ const resolvePackageManagerCommand = (isPackageScript) => {
  * @param {string} scriptName
  * @param {Options | undefined} options
  */
-const getSuggestedScript = async (scriptName, options = { sudo: false }) => {
+const getSuggestedScript = (scriptName, options = { sudo: false }) => {
   const packageScript = findPackageScript(`sku ${scriptName}`);
 
-  const packageManagerCommand = resolvePackageManagerCommand(
-    Boolean(packageScript),
-  );
-
-  const packageOrSkuScript = packageScript
-    ? packageScript
-    : `sku ${scriptName}`;
+  const command = packageScript
+    ? getRunCommand(packageScript)
+    : getExecuteCommand(['sku', packageScript]);
```

Comment:
This looks wrong.

```suggestion
    : getExecuteCommand(['sku', scriptName]);
```

...perhaps?

Reply:
Good catch https://github.com/seek-oss/sku/pull/884/commits/7b5479ad5425278c075ef33444efa0a4a6c1f9f3

Reply:
We need more tests to catch these things. skuba has some nice ones where the output is snapshotted.

Reply:
We have a test case in the `lint-format` suite that tests when a script is found, though it's not testing this behaviour directly. We don't have one for the opposite though.